pkg/compliance/checks/check.go

```diff
@@ -179,6 +180,18 @@ func (c *complianceCheck) Run() error {
 		if c.eventNotify != nil {
 			c.eventNotify(c.ruleID, e)
 		}
+
+		if client := c.StatsdClient(); client != nil {
+			tags := []string{
+				"rule_id:" + e.AgentRuleID,
+				"rule_result:" + e.Result,
+				"rule_framework_id:" + e.AgentFrameworkID,
+				"agent_version:" + e.AgentVersion,
+			}
+			if err := client.Gauge(metrics.MetricChecksStatuses, 1, tags, 1.0); err != nil {
```

Comment:
Is the cardinality of this metric acceptable ? I have honestly no idea

Reply:
I think it is acceptable but I'll remove `rule_framework_id` which is just wasteful. This way, it should be on the same magnitude as the other introduced metrics.